jmclient/jmclient/cryptoengine.py

```diff
@@ -50,6 +55,12 @@ def detect_script_type(script_str):
     raise EngineError("Unknown script type for script '{}'"
                       .format(bintohex(script_str)))
 
+
+def is_extended_public_key(key_str):
+    return any([key_str.startswith(prefix) for prefix in [
```

Comment:
A more robust check would be doing a base58 decoding check here.
But I don' think that has to be part of this PR now.